### ISSUE TITLE
[New build - EXPERIMENTAL] 21w37a: Builder's Requests

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#4177]
This build catches up on some builders most popular requests.

Builders Requests
- Gravity field now has an option for ship-wide gravity.
- Left shift keybind also maps to Right shift.
#4162 Box-delete on decals only toggle inside Build Options
- Docking port shows direction arrows.
- Arch brick added
- Tile triangle brick added 
- Copy-paste and Box delete show symmetry status
- Pick Material in Action Wheel
- Prefabs are editable from the clipboard tab
- Docking Config gate can now merge sub-modules spawned
- Walk speed and Run key re-introduced
- New Build Option: it is now possible to skip some help message (like the "You are stuck" one)
    
 Hotfixes:
 #4165 Copy of children entity render issue
 #4167 Fuel transfert issues
 #3847 Symmetry incorrect on window tile
 Paint quick selection menu fixed
 #4175 RGB controller not working

**Comment by tsunamayo** [comment]
Build is up!

**Comment by 5URG3** [comment]
Awesome! Thanks Tsuna!

**Comment by Servomotor** [comment]
Tsuna is in GodEmperor mode!

**Comment by Servomotor** [comment]
@tsunamayo Sorry to be a pain in the ass, but could we get the thin tile triangle brick. The thinner version is equally as important as the thicker one you blessed us with (also the tooltip says "decorative glowing brick" (lol, I copy paste files for my work as well and do the same thing).

Run speed much better!
RGB working! 
Arch tile is a blessing!

**Comment by Kaiser-Indrasil** [comment]
@tsunamayo Speaking of builder's requests, do you think you could re-introduce the 6th form? I know I'm sounding like a broken record, but this is easily the one shape I'm lacking in the current building engine.

**Comment by Uncle-Ulty** [comment]
> @tsunamayo Speaking of builder's requests, do you think you could re-introduce the 6th form? I know I'm sounding like a broken record, but this is easily the one shape I'm lacking in the current building engine.

I'm with you on this request, Mr. Indrasil

**Comment by Uncle-Ulty** [comment]
> The thinner version is equally as important as the thicker one you blessed us with

Yes, it is! I've been using the glowing triangle painted as black to fulfill this demand. It's a great brick to use on a curved guardrail!



![image](https://user-images.githubusercontent.com/73253472/133290783-7de8e9be-75cd-4f85-902e-7aca2cfc34b9.png)

Omg, this builder's request update was a huge leap on quality of life! Thanks Tsuna!

**Comment by JimStarluck** [comment]
Would it be possible to get an arch variant of the lattice girder too?

**Comment by tsunamayo** [comment]
Glad you like it! @Kaiser-Indrasil yeah I know, it is just that collision on new shape is super time consuming at the moment. One day I should be able to migrate it to something easier to add new shape, but thats low on the list.

**Comment by tsunamayo** [comment]
@JimStarluck I think it will never look good if re-scalable. Would it be fine in a fixe size (but with grid rescale)? Thanks

**Comment by Kaiser-Indrasil** [comment]
> 
> 
> Glad you like it! @Kaiser-Indrasil yeah I know, it is just that collision on new shape is super time consuming at the moment. One day I should be able to migrate it to something easier to add new shape, but thats low on the list.

@tsunamayo I see. Still, I'm hoping it would come sooner rather than later because quite a lot of people are waiting on it.

**Comment by JimStarluck** [comment]
@tsunamayo I can work with that.

**Comment by Arutus** [comment]
can we still eventually expect all of the glass blocks we discussed earlier in our "discussions" post?
